strax/processing/peak_building.py

```diff
@@ -125,19 +125,26 @@ def store_downsampled_waveform(p, wv_buffer):
     if downsample_factor > 1:
         # Compute peak length after downsampling.
         # Do not ceil: see docstring!
+        #Doing this is a loop is not allowed by numba
```

Comment:
I guess you mean looping over the field names of the structured array right? Yes this is somewhat annoying...

Reply:
Yes, making 1 bigger buffer to hold all 3 does not work